roles/docker/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: "Setting Docker Facts"
+  set_fact:
+    docker_storage_block_device: "{{ docker_storage_block_device | default(default_docker_storage_block_device) }}"
+    docker_storage_volume_group: "{{ docker_storage_volume_group | default(default_docker_storage_volume_group) }}"
+
+- name: "Install Docker"
```

Comment:
yes, this is what should happen for contrib and what we have been moving towards in CASL, implementation is just a bit behind